.github/workflows/db-migrations.yml

```diff
@@ -6,6 +6,7 @@ on:
     - .github/workflows/db-migrations.yml
     - lib/bin/create-docker-databases.js
     - lib/model/migrations/**
+    - lib/model/migrations-post-knex/**
```

Comment:
TODO rename these.  Don't put new migrations in `/migrations/` in case that creates conflicts with custom/edited migrations in 3rd party deployments.